examples/typescript/yourCoin/moonCoin/moonCoin.json

```diff
@@ -0,0 +1,16 @@
+{
+  "function_id": "0x1::code::publish_package_txn",
+  "type_args": [],
+  "args": [
+    {
+      "type": "hex",
+      "value": "0x084d6f6f6e436f696e0100000000000000004032344143313145423932453634313844304231433438373434313939313043444631453544454532383836333335383231353241353435444145423932454533aa011f8b08000000000002ff3d8db10ec2300c44777f05ca4ec28cc48090d8f882aa426962daa8c48ee2b4fc3e89689117dfdd3bbb4bd6cd76c41ec8463c5c0eeac14c370ea460c52c81a999275d470174d6fb8c22283dec60cb9f2df398903c920b28fa9a0acb3dd7a31fce730f63280d9c4a497236a6ca6919b4e3686c238f6f3bc8b63acea82ba020e3da4ad10622ac5a96c187dcac1f197945f3da9f6cf5bf56f005ed2121f8de00000001096d6f6f6e5f636f696eac011f8b08000000000002ff654e4b0ac23010dde7144317d2826b1751dcb8d61ea1c4765a82c944f241b0e4ee26412bd559bd79f33ea3cd1014c2d9183a19499ceb84ba3e419819a471de86de2f0298232bfc18082449dfe992503ba4012d878d9313a16ddef63ce2ee8deb462b343e8cbda50e4162c2a1d4709e53a450f289874fcbb15ecce58792bd5d71d72a8b21ababbf4bdb5e7ec8dd7a1d8572f8a59a7d819145f602f36168b21101000000000300000000000000000000000000000000000000000000000000000000000000010e4170746f734672616d65776f726b00000000000000000000000000000000000000000000000000000000000000010b4170746f735374646c696200000000000000000000000000000000000000000000000000000000000000010a4d6f76655374646c696200"
```

Comment:
We should not commit these .json files because they have the address in them, and will always change.